UKHomeOffice/sgmr-service
# SPCR-411 Onboarding notification banner persisting on people page

### AC / Description of changes
When you fill out add a new person in the onboarding and complete the onboarding, if you click on the people tab from the dashboard, the 'person successfully added' banner is still there until you click off it and go back.

### To Test
- Click start on the onboarding banner
- Complete up until the 'do you want to add people' page
- Click yes
- Fill out a person
- > You should be taken back to the 'do you want to add people' page
- > You should see a banner saying 'person successfully added'
- Click no and continue
- From the dashboard click on the people tab
- The banner should not persist here

### Notes

To make testing easier, rather than clearing your account of all voyages, people, and pleasurecraft you can set the onboarding state to always be true

- on Dashboard.jsx
- within the useEffect starting on line 73
- add setShowOnboarding(true)

```
useEffect(() => {
    if (userData && Object.keys(voyageStatusCounts).length > 0) {
      const totalVoyages = Object.values(voyageStatusCounts).reduce((a, b) => a + b);
      setShowOnboarding(userData.people.length === 0 && userData.vessels.length === 0 && totalVoyages === 0);
    }
    setShowOnboarding(true);
  }, [userData, voyageStatusCounts, setShowOnboarding]);
  ```

---

**Remember to merge to feature branches not `master`.**
